README.md

```diff
@@ -22,150 +22,3 @@ Please go to the <a href="https://github.com/YurisCodingClub/html-tag-quiz/discu
-
-For example:
-
-```
-git clone https://github.com/this-is-you/dev-cv.git
-```
-
-where `this-is-you` is your GitHub username. Here you're copying the contents of the dev-cv repository on GitHub to your computer.
-
-### Create a branch
-
-Change to the repository directory on your computer (if you are not already there):
-
-```
-cd dev-cv
-```
-
-Open the project in Visual Studio Code (or your favorite code editor):
-
-```
-code .
-```
-
-Run the project:
-
-Right click on the `index.html` file and choose
-
-```
-Open with Live Server
-```
-
-Now create a branch using the `git checkout -b` command:
-
-```
-git checkout -b your-new-branch-name
-```
-
-For example:
-
-```
-git checkout -b name-of-issue
-```
-
-### Make necessary changes and commit those changes
-
-<img align="right" width="450" src="https://docs.google.com/uc?export=download&id=16K70wMG4sZk2MzqwkAowkX06EpI4dMKZ" alt="git status" />
-
-If you go to the project directory and execute the command `git status`, you'll see there are changes.
-
-Add those changes to the branch you just created using the `git add` command:
-
-```
-git add filename
-```
-
-Now commit those changes using the `git commit` command:
-
-```
-git commit -m "Name of issue"
-```
-
-#### Commit message
-
-There are different ways of writing a commit message. For more reference checkout the
-article about [How to write a good commit message](https://dev.to/chrissiemhrk/git-commit-message-5e21).
-
-For this project we are going to use following style:
-
-**type-of-issue- + number-of-issue[action you took]**
-
-Practical example:
-
-`feature-11[Fix typo in README.md]`
-
-### Push changes to GitHub
-
-Push your changes using the command `git push`:
-
-```
-git push origin -u <add-your-branch-name>
-```
-
-replacing `<add-your-branch-name>` with the name of the branch you created earlier.
-
-### Submit your changes for review
-
-If you go to your repository on GitHub, you'll see a `Compare & pull request` button. Click on that button.
-
-<img style="float: right;" src="https://docs.google.com/uc?export=download&id=1FMXVHEhxBxeSFWsJsR53ddE8GmyAlMR4" alt="create a pull request" />
-
-Now submit the pull request.
-
-<img style="float: right;" src="https://docs.google.com/uc?export=download&id=1r6VFuraC-5l9PF4JKn9A4Rhmm-4VelI7" alt="submit pull request" />
-
-Soon I'll be merging all your changes into the master branch of this project. You will get a notification email once the changes have been merged.
-
-## Where to go from here?
-
-Congrats! You just completed the standard _fork -> clone -> edit -> pull request_ workflow that you'll encounter often as a contributor!
-
-Celebrate your contribution and share it with your friends and followers on your social media accounts.
-
-Now let's get you started with your next contribution, wether on this or any other great open source projects on GitHub.
-
-Have fun 👻!
```

Comment:
This sentence got lost somewhere 😄 I added it to the discussion page!

Reply:
I'm sorry, I thought this was the sentence that was supposed to be removed :(

Reply:
No worries, my sentences was not very well formulated, as for 

> remove ... as well

was meant as **transferred with the other content**. 😉

Reply:
Ahh, I get it now :)